src/hosting/normalizedHostingConfigs.ts

```diff
@@ -1,31 +1,38 @@
-const _ = require("lodash");
+import * as _ from "lodash";
 
-function _filterOnly(configs, onlyString) {
+type HostingConfig = { site: string; target: string };
+
+/**
+ * @param configs
+ * @param onlyString
+ * @return {HostingConfig[]}
+ */
+function _filterOnly(configs: HostingConfig[], onlyString: string): HostingConfig[] {
   if (!onlyString) {
     return configs;
   }
 
-  var onlyTargets = onlyString.split(",");
+  let onlyTargets = onlyString.split(",");
   // If an unqualified "hosting" is in the --only,
   // all hosting sites should be deployed.
   if (_.includes(onlyTargets, "hosting")) {
     return configs;
   }
 
   onlyTargets = onlyTargets
-    .filter(function(anOnly) {
-      return anOnly.indexOf("hosting:") === 0;
-    })
-    .map(function(anOnly) {
-      return anOnly.replace("hosting:", "");
-    });
+    .filter((anOnly) => anOnly.startsWith("hosting:"))
+    .map((anOnly) => anOnly.replace("hosting:", ""));
 
-  return configs.filter(function(config) {
-    return _.includes(onlyTargets, config.target || config.site);
-  });
+  return configs.filter((config: HostingConfig) =>
+    _.includes(onlyTargets, config.target || config.site)
+  );
 }
 
-module.exports = function(options) {
+/**
+ * @param options
+ * @return {any[] | HostingConfig[]}
```

Comment:
This JSDoc doesn't match the signature (also I'd say we should just remove the doc comment dince it doesn't tell you anything the function signature doesn't show)

Reply:
ok i'll remove this.
(it generate by lint --fix)

Reply:
`lint --fix` adds this return type? I will dig into that as it shouldn't in TS...